setup.py

```diff
@@ -80,25 +80,7 @@ def finalize_options(self):
       unit-tested window mangers around.
 """
 
-if '_cffi_backend' in sys.builtin_module_names:
```

Comment:
I think that this part should stay, this is more for PyPy to make sure that we don't try to upgrade the cffi version (since cffi is bundled in PyPy)